migrations/0005_migrations.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+Sequel.migration do
+  transaction
```

Comment:
You'll need to add a foreign key to 0004_migrations for a sku_id to both node_resources_projects and db_resources_projects so that we can call project.db/node_resources_projects and get the sku id.